src/propcache/_helpers_py.py

```diff
@@ -46,12 +46,12 @@ def __init__(self, wrapped: Callable[..., _T]) -> None:
         self.name = wrapped.__name__
 
     @overload
-    def __get__(  # pragma: no cover
+    def __get__(  # pragma: no branch
```

Comment:
Where did you see partial coverage coming from?

Reply:
I only saw it on the report.

https://app.codecov.io/gh/aio-libs/propcache/blob/master/src%2Fpropcache%2F_helpers_py.py